identifiers/country-bm.csv

```diff
@@ -0,0 +1,2 @@
+id,name
+ocd-division/country:bm,Bermuda

```

Comment:
This is a territory of the UK, so I don't think it should be a separate country.